### ISSUE TITLE
Unable to type four hex digit utf codes in acme on mac

### ISSUE THREAD

**Issue by sirnewton01** [9fans/plan9port#224]
I am following the manual page in keyboard(7) to try and type in different hex codes for runes on Mac OS (High Sierra) in acme. I can press and release the alt key to type in the quick codes (e.g. M^ for the musical notation for natural). However I can't seem to hold down the alt key and type the four hex digit codes for items without the quick codes. Perhaps the documentation needs to be updated or maybe I'm just doing it wrong. I'm not sure.

Thanks

**Comment by jxy** [comment]
> Any 16-bit rune can be typed using a compose key followed by several other keys.

Do not hold the alt key down.

**Comment by sirnewton01** [comment]
Thanks, I tried holding the alt key down and not holding it down. When I try to input 266e (music natural) from the lib/keyboard file it pauses for the 2 and 6 but then when I type the second 6 it starts just outputting what I type instead of the rune. I noticed that I can get the sharp using the quick code M^ by not holding down the alt key but I can't seem to input anything that doesn't have one, like 2423 (open box, visible space).
